packages/push-platform-notifications-controller/src/types/firebase.ts

```diff
@@ -0,0 +1,46 @@
+export declare type Messaging = {
```

Comment:
Do we need to use `declare` in this file?

Reply:
No. Left over from the copy. Fixed.